### PR TITLE
Generic Focus wrapper component

### PR DESCRIPTION
**Pull request by codeuri**
Worked on:
- Small refactoring of existing component
- Dynamic color for component/added color props validation/added default color
- Dynamic size for component/added size props validation/added default size
- Dynamic opacity for component/added default opacity
- Component styles
- Tests for the above features.
- Stories update

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit ffbb2037f06fc0d11701feb8251d5ff8a2f2315a

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5c7c516c4f630c000806247f

**Comment by TimMansell** [comment]
Additional tasks

- Need ability to add non-padded content

File: components/focus-wrapper/index.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    &.purple {
+      fill: rgb(var(--col-purple-mid-2));
+    }
+
+    &.pink {
+      fill: rgb(var(--col-pink-mid-2));
+    }
+
+    &.semi-opaque {
+      opacity: .33;
```
Comment (TimMansell) [inline components/focus-wrapper/index.css]: Update to .35

File: components/focus-wrapper/index.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+  }
+
+  &__bottomRight {
```
Comment (TimMansell) [inline components/focus-wrapper/index.css]: kebab-case class name

File: components/focus-wrapper/index.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
+.card-focus {
+  display: flex;
+  position: relative;
+  padding: 65px 54px 72px 95px;
+
+  &__topLeft,
```
Comment (TimMansell) [inline components/focus-wrapper/index.css]: kebab-case class name

File: components/focus-wrapper/index.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  &__bottomRight {
```
Comment (TimMansell) [inline components/focus-wrapper/index.css]: kebab-case class name

File: components/focus-wrapper/FocusWrapper.vue [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    color: {
+      type: String,
+      default: '',
+      validator: color => ['', 'navy', 'teal', 'yellow', 'emerald', 'orange', 'green', 'purple', 'pink'].indexOf(color) > -1,
+    },
+    semiOpaque: {
+      type: Boolean,
+      default: false,
+    },
+    size: {
```
Comment (gepifanio) [inline components/focus-wrapper/FocusWrapper.vue]: When we see size as a prop could be in doubt what is this exactly, would be better to be more specific such as:
`iconSize`
`iconColor`

**Comment by TimMansell** [comment]
I'm happy to merge.  @meligatt, are you fine to review and merge tomorrow?

**Pull request merged**